src/NUnitEngine/nunit.engine.tests/Internal/TcpChannelUtilsTests.cs

```diff
@@ -33,24 +33,24 @@ namespace NUnit.Engine.Internal.Tests
 	/// Summary description for RemotingUtilitiesTests.
```

Comment:
I'm going to guess TcpChannelUtilsTests a.k.a. ServerUtilityTests was called RemotingUtilitiesTests at some point in a past life. 😆 This comment is useless and could go - but don't worry about it if someone comes back to merge this first!